packages/block-editor/src/store/selectors.js

```diff
@@ -2299,12 +2321,12 @@ export const __experimentalGetParsedPattern = createRegistrySelector(
 					__unstableSkipMigrationLogs: true,
 				} ),
 			};
-		}, getAllPatternsDependants )
```

Comment:
At some point we should change `__experimentalGetParsedPattern` so it doesn't depend on all patterns...